ozaria/site/components/sign-up/PageEducatorSignup/common/signUpValidations.js

```diff
@@ -13,6 +13,14 @@ const uniqueEmail = async (email) => {
   return true
 }
 
+const uniqueName = async (name) => {
```

Comment:
a better name could be isUniqueName?

Reply:
lol we use uniqueEmail above. so i keep in sync